feat: Document <meta name="sentry-trace"> (#2279)
getsentry/sentry-docs: src/platforms/javascript/common/performance.mdx

_NOTE:_ You need to make sure your web server CORS is configured to allow the `sentry-trace` header. The configuration might look like `"Access-Control-Allow-Headers: sentry-trace"`, but the configuration depends on your set up. If you do not allow the `sentry-trace` header, the request might be blocked.

### Connecting Backend and Frontend Traces

When instrumenting both frontend and backend with Sentry (Node.js or other

> supported platform), it is possible to connect the pageload transaction on the

Reviewer: ```suggestion
supported platforms), it is possible to connect the page load transaction on the
```

> When instrumenting both frontend and backend with Sentry (Node.js or other
> supported platform), it is possible to connect the pageload transaction on the
> frontend with the trace that originated the page in the backend using a special
> `<meta>` tag in the `<head>` of the HTML document.

Reviewer 2: ```suggestion
When instrumenting both frontend and backend with Sentry (Node.js or other
supported platform), you can connect the pageload transaction on the
frontend with the trace that originated the page in the backend by using a special
`<meta>` tag in the `<head>` of the HTML document.
```

Author: Applied, thanks!

> Include in the rendered page a meta tag like so:

Reviewer 2: ```suggestion
Include a `<meta>` tag in the rendered page as in this example:
```

```html
<html>
<head>
  <meta name="sentry-trace" content="{{ span.toTraceparent() }}">
  <!-- ... -->
```

> The `content` attribute should have a special string generated by the Sentry
> SDK. The exact syntax depends on your platform and template library.

Reviewer 2: ```suggestion
The Sentry SDK will generate a special string for the `content` attribute. The precise syntax varies by platform and template library.
```

<Alert level="info" title="Note"><markdown>

In this particular case, the `<meta>` tag is used to propagate tracing metadata,

> replacing the use of the equivalent `sentry-trace` HTTP header described

Reviewer: ```suggestion
replacing the equivalent `sentry-trace` HTTP header described
```

Author: Applied

> In this particular case, the `<meta>` tag is used to propagate tracing metadata,
> replacing the use of the equivalent `sentry-trace` HTTP header described
> earlier. That is because browsers do not provide any API for reading the
> response headers of the current page.

Reviewer 2: ```suggestion
In this example, the `<meta>` tag is used to propagate tracing metadata,
replacing the use of the equivalent `sentry-trace` HTTP header described
earlier. This is because browsers do not provide any API for reading the
response headers of the current page.
```

_NOTE:_ You need to make sure your web server CORS is configured to allow the `sentry-trace` header. The configuration might look like `"Access-Control-Allow-Headers: sentry-trace"`, but the configuration depends on your set up. If you do not allow the `sentry-trace` header, the request might be blocked.

### Connecting Backend and Frontend Traces

When instrumenting both frontend and backend with Sentry, you can connect the
`pageload` transaction on the frontend with the trace that originated the page
in the backend by including a special `<meta>` tag in the `<head>` of the HTML
document.

Example:

```html
<html>
<head>
  <meta name="sentry-trace" content="{{ span.toTraceparent() }}">
  <!-- ... -->
```

The `name` attribute must be the string `"sentry-trace"` and the `content`
attribute must be generated by your backend's Sentry SDK using
`span.toTraceparent()` and inserted in the HTML page.

The `span` reference is either the transaction that rendered the page or any of
its child spans.

For the correct association of transactions in a trace, every HTTP request
should render a new and unique value of `traceparent` in the `content`

> attribute.

Author: Rephrased. Thanks for all the feedback, Katie!
